lib/features/children/overview/pages/models/edit_allowance_success_uimodel.dart

```diff
@@ -1,5 +1,7 @@
 class EditAllowanceSuccessUIModel {
-  const EditAllowanceSuccessUIModel({this.amountWithCurrencySymbol});
+  const EditAllowanceSuccessUIModel(
+      {this.amountWithCurrencySymbol, this.onClickButton});
```

Comment:
I think here as well we have a small text change: "Your child(ren) will receive $15 each month. You can edit this amount any time."